CMakeLists.txt

```diff
@@ -33,75 +33,108 @@ find_package(gflags REQUIRED)
 ### tomlplusplus (for configuration files)
 FetchContent_Declare(
   tomlplusplus
-  GIT_REPOSITORY https://github.com/marzer/tomlplusplus.git
-  GIT_TAG        4b166b69f28e70a416a1a04a98f365d2aeb90de8 # v3.2.0
-  GIT_PROGRESS   TRUE
+  GIT_REPOSITORY    https://github.com/marzer/tomlplusplus.git
+  GIT_TAG           4b166b69f28e70a416a1a04a98f365d2aeb90de8 # v3.2.0
+  GIT_PROGRESS      TRUE
+  FIND_PACKAGE_ARGS
 )
 FetchContent_MakeAvailable(tomlplusplus)
 
 ### glog
-FetchContent_Declare(
-  glog
-  GIT_REPOSITORY https://github.com/google/glog.git
-  GIT_TAG        96a2f23dca4cc7180821ca5f32e526314395d26a
-  GIT_PROGRESS   TRUE
-)
-FetchContent_MakeAvailable(glog)
-
-# These glog executable targets still generate warnings - disable warnings for
-# them explicitly
-target_compile_options(demangle_unittest PRIVATE "-w")
-target_compile_options(logging_unittest PRIVATE "-w")
-target_compile_options(stl_logging_unittest PRIVATE "-w")
-target_compile_options(symbolize_unittest PRIVATE "-w")
-target_compile_options(utilities_unittest PRIVATE "-w")
+find_package(glog)
+if (NOT glog_FOUND)
```

Comment:
As you yourself spotted yesterday, this isn't needed when using FIND_PACKAGE_ARGS

Reply:
Yep! Other way round here though, it needed the check because those warning options (not entirely sure we need them but didn't want to make this any more complicated) aren't compatible with a `find_package` version of glog.